src/core/core.scale.js

```diff
@@ -1252,9 +1252,13 @@ export default class Scale extends Element {
         const labelPadding = toPadding(optsAtIndex.backdropPadding);
         const height = labelSizes.heights[i];
         const width = labelSizes.widths[i];
+        const metrics = this.ctx.measureText(label);
```

Comment:
Why do we need to re-measure the text here?

Reply:
For rotated labels the x & y are completely off - I don't know why - but they're not calculated using measureText(), presumably because at the time it was written measureText() didn't return consistent results across browsers.